tools/bootstrap-librdkafka.sh

```diff
@@ -25,6 +25,8 @@ if [[ $INSTALLDIR != /* ]]; then
     INSTALLDIR="$PWD/$INSTALLDIR"
 fi
 
+rm -rf ${BUILDDIR}
```

Comment:
Need quotes around the variable.

Why are you removing the builddir though? The code below has logic to use an existing build directory.

Reply:
removed